commercetools-models/src/main/java/io/sphere/sdk/orderedits/commands/OrderEditApplyCommand.java

```diff
@@ -0,0 +1,50 @@
+package io.sphere.sdk.orderedits.commands;
+
+import com.fasterxml.jackson.databind.JavaType;
+import io.sphere.sdk.client.HttpRequestIntent;
+import io.sphere.sdk.commands.CommandImpl;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.orderedits.OrderEdit;
+
+import static io.sphere.sdk.http.HttpMethod.POST;
+
+public final class OrderEditApplyCommand extends CommandImpl<OrderEdit> {
+
+    private final String id;
+
+    private final Long resourceVersion;
+
+    private final Long editVersion;
+
+    private OrderEditApplyCommand(final String id, final Long resourceVersion, final Long editVersion) {
```

Comment:
Here just for clarity reason, name the params as `orderEditId`, `orderEditVersion`, and `orderVersion` in this order,
also add a factory method with parameters `Versioned<OrderEdit>` and `orderVersion`